test/system/alpha/ordered_list_test.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "application_system_test_case"
+
+class IntegrationAlphaOrderedListTest < ApplicationSystemTestCase
```

Comment:
Since there's no Javascript behavior associated with these list components, we probably don't need system/integration tests for them.